src/app/visual/theming/theming-demo.component.html

```diff
@@ -97,5 +97,15 @@ <h2 class="sky-font-heading-5">Heading 5 (by CSS class)</h2>
     <div class="sky-font-emphasized">Emphasized text</div>
     <div class="sky-font-deemphasized">De-emphasized text</div>
     <div class="sky-headline">Headline</div>
+    <div id="placeholder-test">
```

Comment:
Since you're adding styles for all inputs and not just input box, wouldn't it be sufficient to take a screenshot of a plain `input` element? I would expect any screenshots of placeholder text inside an input box would be in the same library as input box.

Reply:
It actually does make a difference here. 

The key is that we have styles that change the text font size to 16px for inputs inside of an input box but we leave it at 15px for those that are outside of it. So before this change the main text of inputs in an input box was 16px but the placeholder text was 15px. This was caught during the lookup work when trying to line the tokens up with the text. 

We could add a screenshot to the input box library and just do a standard input but I did this here as we wouldn't have caught this discrepancy without seeing the input box style here. 

This is an odd one because it is a case where the input box styles caused things to be different but the fix was in `theme` and so it gets tricky to test.

Reply:
Hmm...so if input box is setting the input's text size to 16px, I would expect input box to also set the input's placeholder text size to 16px. I'm also not sure of the intent of `font-size: inherit` since that's going to use the font size of the input's parent element.

Reply:
I pulled the font size portion directly from the modern styles for the deemphasized text.

https://github.com/blackbaud/skyux-theme/blob/d5c73178a82c7793c613cfd6aab77b25b444f87c/src/app/public/styles/themes/modern/_type.scss#L34

I know nothing of the history there.

Reply:
My change here is just setting the placeholder text styles to the same as the `sky-font-deemphasized` styles in modern theme. 

In default we did this with a mixin but it appeared that we don't have a mixin for these styles any more in modern. (I assumed deliberately as we've gotten away from mixins more here recently)

Reply:
I didn't feel like we needed to make a change to input box's styles as the base deemphasized style already used the `inherit` and so it wasn't really needed if we made this change for the two to match.

Reply:
I believe that's all true and correct, but @Blackbaud-ToddRoberts can confirm the intent about avoiding mixins.

Reply:
> I pulled the font size portion directly from the modern styles for the deemphasized text.
> 
> https://github.com/blackbaud/skyux-theme/blob/d5c73178a82c7793c613cfd6aab77b25b444f87c/src/app/public/styles/themes/modern/_type.scss#L34
> 
> I know nothing of the history there.

I think `inherit` makes sense there because the deemphasized class is typically used on an element like a `div` or `span` that will inherit its font size from its parent, but `input` elements don't inherit their font size from their parent element. For instance, if you have this HTML:

```
<div style="font-size: 48px">
  <input type="text" value="hello">
</div>

Reply:
So are we wanting to set the base at 15px then and then overwrite that in the input box styles?

Reply:
You are correct that inputs don't normally inhert their size. Most of our inputs the size comes in via the `sky-form-control` class which sets it to 15px (unless it is overwritten by input box).

The `inherit` that I had worked because the placeholder is considered a child in the shadow-dom and so it inherited from that class on the `input`.

Reply:
Saying that though - I get what you are saying that without the `sky-form-control` class it would inherit up the tree (i.e. your 48px div) and we probably wouldn't want that.

Reply:
Lets discuss in flex as the fact that input box is the one that overwrites the font size at all is strange to me.